mimic/model/clb_objects.py

```diff
@@ -402,10 +432,9 @@ def add_node(self, node_list, lb_id, current_timestamp):
                         return (resource, 413)
 
             # If there were no duplicates
-            new_nodeCount = self.lbs[lb_id]["nodeCount"] + len(nodes)
+            new_nodeCount = len(self.lbs[lb_id].nodes) + len(nodes)
             if new_nodeCount <= self.node_limit:
-                self.lbs[lb_id]["nodes"] = self.lbs[lb_id]["nodes"] + nodes
-                self.lbs[lb_id]["nodeCount"] = new_nodeCount
+                self.lbs[lb_id].nodes = self.lbs[lb_id].nodes + nodes
```

Comment:
Just noticing this could use `+=` to be a little clearer, since `self.lbs[lb_id].nodes` is a complex enough expression I had to read it twice to make sure it was the same :).